testing/runner_test.ts

```diff
@@ -22,7 +22,7 @@ test(async function getMatchingUrlsRemote(): Promise<void> {
 
 test(async function getMatchingUrlsLocal(): Promise<void> {
   const urls = await getMatchingUrls(
-    ["fmt/*_test.ts"],
+    [join("fmt", "*_test.ts")],
```

Comment:
Doh! `fmt` is specified in `TEST_ROOT_PATH`, change this to just `["*_test.ts"],`.

There's a reason this works on unix and I've found lots of other issues. Let's just get this green and I'll see what I can improve in #604.